File: tree-sitter-julia/test/corpus_smoke/02_functions.jl
function add(a::Int, b::Int)::Int
    return a + b
end

mul(a, b) = a * b

function variadic(args...; scale=1, kwargs...)
    sum(args) * scale
end

square = x -> x^2
composed = square ∘ add

println(add(1, 2), " ", variadic(1, 2, 3; scale=2), " ", composed(1, 2))

File: tree-sitter-lua/test/corpus_smoke/02_functions.lua
local function add(a, b)
  return a + b
end

local mul = function(a, b) return a * b end

local function variadic(...)
  local args = { ... }
  return select("#", ...), args
end

print(add(1, 2), mul(3, 4), variadic("x", "y"))
